### PR TITLE
Homework from wednesday

### PR DESCRIPTION
**Pull request by AleksKuzmin**
_If you had no problems you can just leave it empty_

- How difficult did you find this (out of 10)? (0 being no problems at all, 10 being impossible)
- 7 out 10
- Was there anything that you struggled with?
- I still struggle with using a loop inside of a loop
- Is there any particular code you want me to look into?
Multiply and Sum and also longest string ( I only manage to compare with a second string)
- Is there anything that you'd like some further information on?
- logic behind loop inside a loop
- Roughly how long did it take?
- 2.5- 3hours

### CODE REVIEW

**Review by patmifsud — APPROVED** [review]
Hey Aleks, your code looks great 👍. Really neat and efficient. 
Congrats on getting through the homework. 
I've added a few comments here and there, let us know if you want to discuss anything

File: Aleksandr Kuzmin/arrays/js/homework.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+  if( num1 > num2 ) {
+    return num1;
+
+  }if(num1 < num2){
+    return num2;
+
+  }
+}
+console.log(`${maxOfTwoNumbers(3,5)}`);
```
[the remark this answers is not in the document]
Comment (AleksKuzmin) [reply Aleksandr Kuzmin/arrays/js/homework.js:26]: I thought, I would be calling a function at the same time

[the remark this answers is not in the document]
Comment (patmifsud) [reply Aleksandr Kuzmin/arrays/js/homework.js:26]: At the same time as a string? like ` `hello this is the answer ${maxOfTwoNumbers(3,5) }` `?
In that case yeah, you would want backticks and ${ } 👍

**Comment by AleksKuzmin** [comment]
yep, cool! thank you!

**Comment by AleksKuzmin** [comment]
Also, could you please help me with sum and multi tasks as I think the
solution. As my solution is not efficient to me as it only works if I add
numbers to the string. I think I need another loop for it but I
didnt manage to fit one in. Thanks Alex

**Comment by patmifsud** [comment]
> Also, could you please help me with sum and multi tasks as I think the solution. As my solution is not efficient to me as it only works if I add numbers to the string. I think I need another loop for it but I didnt manage to fit one in. Thanks Alex
> […](#)

Sure, it might be easier to go through in zoom, just ping me 👍

**Pull request merged**